gravitee-apim-gateway/gravitee-apim-gateway-tests-sdk/src/test/resources/gravitee-policy-fake/.circleci/config.yml

```diff
@@ -0,0 +1,35 @@
+version: 2.1
+
+# this allows you to use CircleCI's dynamic configuration feature
```

Comment:
is it necessary to have a dedicated circleci config 🤔 They will be built manually from what I understand

Reply:
true, I forgot to remove it